### PR TITLE
fix: generate image random tag

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Generate image random tag for scm triggered wfr

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/server/handler/v1alpha1/webhook.go [hunk 1]
```diff
@@ -184,10 +185,14 @@ func createWorkflowRun(tenant string, wft v1alpha1.WorkflowTrigger, data *scm.Ev
 		return err
 	}
 
+	if tag == "" {
```
Comment (supereagle) [inline pkg/server/handler/v1alpha1/webhook.go]: Should add this logic in https://github.com/caicloud/cyclone/pull/1145/files#diff-4db6e7deb63057dec099088e5e228cfbR160.

```
	suffix := rand.String(5)
	name := fmt.Sprintf("%s-%s", wfName, suffix)
	alias := name
	if tag != "" {
		alias = tag
	} else {
		tag = suffix
	}
```

**Comment by zhujian7** [comment]
@supereagle PTAL

**Pull request merged**